### DISCUSSION TITLE
Weird Issues After Following Official Installation Guide

### DISCUSSION THREAD

**Discussion by ngx3** [searx/searx#2990] [Installation issue: Official wiki]
So, I'm getting a bunch of issues and errors with this installation, I'm following the [official guide](https://searx.github.io/searx/admin/installation.html), and after fixing several issues myself others appear. I do plan to make the instance public and therefore using Filtron and Morty.

> INFO:searx:merge the default settings ( /usr/local/searx/searx-src/searx/settings.yml ) and the user setttings ( /etc/searx/settings.yml )
> INFO:searx:Initialisation done
> INFO:searx.plugins.https_rewrite:29 rules loaded
> INFO:searx.shared:Use shared_simple implementation
> INFO:searx.search:max_request_timeout=None
> DEBUG:searx.webapp:static directory is /usr/local/searx/searx-src/searx/static
> DEBUG:searx.webapp:templates directory is /usr/local/searx/searx-src/searx/templates
> DEBUG:searx.webapp:starting webserver on 127.0.0.1:8888
>  * Serving Flask app "webapp" (lazy loading)
>  * Environment: production
>    WARNING: This is a development server. Do not use it in a production deployment.
>    Use a production WSGI server instead.
>  * Debug mode: on
> Traceback (most recent call last):
>   File "searx/webapp.py", line 1208, in <module>
>     run()
>   File "searx/webapp.py", line 1140, in run
>     threaded=True
>   File "/usr/local/searx/searx-pyenv/lib/python3.6/site-packages/flask/app.py", line 990, in run
>     run_simple(host, port, self, **options)
>   File "/usr/local/searx/searx-pyenv/lib/python3.6/site-packages/werkzeug/serving.py", line 982, in run_simple
>     s.bind(server_address)
> OSError: [Errno 98] Address already in use
> 

Also, I'm getting that I can't change the public address/URL of my instance without removing everything and running the script again?

On Ubuntu 18.04 using Nginx 1.21.3.

**Comment by unixfox** [comment]
You just already have something else running on the port 8888

**Reply by ngx3** [reply]
uwsgi on 8888 is the only thing.

**Reply by unixfox** [reply]
I'm just explaining to you what the error `[Errno 98] Address already in use` is.

You probably have another searx installation somewhere else that is already running, I don't really know...

**Reply by ngx3** [reply]
> 'm just explaining to you what the error [Errno 98] Address already in use is

I know. I'm just making it clear that I only have that running on port 8888.

> You probably have another searx installation somewhere else that is already running, I don't really know...

I've tried everything I could think of, I'm all out of ideas. That's why I came here.
